src/Common/test-runtime/XUnit.Runtime.depproj

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I'm putting things like this into under /external directory. I'm fine with it here for now until that comes online though.